dali/server/daserver.cpp

```diff
@@ -475,6 +475,7 @@ int main(int argc, const char* argv[])
 
         for (unsigned i=1;i<(unsigned)argc;i++) {
             if (streq(argv[i],"--daemon") || streq(argv[i],"-d")) {
+                i++; // swallowing <instanceName> after --daemon|-d
```

Comment:
This will work fine.  Technically this is part of HPCC-26750 but it is up to @richardkchapman how we include it.

You might want to add some error checking for instanceName existing. ex:  i + 1 < argc to ensure the next argument exists, !strncmp(instanceName,"-",1) to check that the value following the daemon flag isn't another flag.